internal/types.go

```diff
@@ -105,6 +113,32 @@ type MediaIds struct {
 	Ids []string `json:"ids"`
 }
 
-type Env struct {
-	db *gorm.DB
+func getMediaType(id string) string {
+	if id[0:1] == "m" {
+		return "movie"
+	}
+	return "tv"
+}
```

Comment:
Should probably handle "t" as well, for when we get people. Throw an easy to understand error if it's not m or t 🗡️

Reply:
Done